Standups-StackBased/Standups/StandupsApp.swift

```diff
@@ -9,8 +9,8 @@ struct StandupsApp: App {
       //     dependencies for the duration of the test (e.g. the data manager). We do not really
       //     recommend performing UI tests in general, but we do want to demonstrate how it can be
       //     done.
-      if ProcessInfo.processInfo.environment["UITesting"] == "true" {
-        UITestingView()
+      if let testName = ProcessInfo.processInfo.environment["UITest"] {
+        UITestingView(testName: testName)
```

Comment:
I'm not having the integration test target set an env var to describe which test is running so that we can switch on it and decide what dependencies to override.